docs/Reference/Server.md

```diff
@@ -374,6 +375,28 @@ The possible values this property may have are:
   const fastify = require('fastify')({logger: customLogger});
   ```
 
+### `createRequestLogMessage`
+<a id="create-request-log-message"></a>
+
++ Default: `true`
+
+When logging is enabled via `disableRequestLogging` parameter, users can 
+implement the `createRequestLogMessage` function to customize their log management 
+system.  As a result, this setting allows users to produce a customizable log message 
+catered to their projects' needs, circumventing the default format. 
+
+Example:
+
+```js
+  if (disableRequestLogging === false) {
+        if (typeof createRequestLogMessage === 'function') {
+          childLogger.info({ req: request }, createRequestLogMessage(req, 'customMessage'))
+        } else {
+          childLogger.info({ req: request }, 'incoming request')
+        }
+      }
```

Comment:
The indentation is not at all consistent in this block. I am also still unclear why this "example" is showing what looks like Fastify's internal implementation of the feature. It should be showing how to use it as user of Fastify.

Reply:
> The indentation is not at all consistent in this block.

Hi @jsumners, no problem, I'll make sure to re-adjust the indentation.

Reply:
>  I am also still unclear why this "example" is showing what looks like Fastify's internal implementation of the feature. It should be showing how to use it as user of Fastify.

Ok sounds good, I'll make sure to change that so it's from the user's perspective.